aws/resource_aws_rds_cluster_instance_test.go

```diff
@@ -852,6 +1044,71 @@ resource "aws_db_parameter_group" "bar" {
 
 func testAccAWSClusterInstanceConfigPerformanceInsightsEnabledAuroraMysql1(rName string) string {
 	return fmt.Sprintf(`
+resource "aws_rds_cluster" "test" {
+  cluster_identifier  = %[1]q
+  database_name       = "mydb"
+  engine              = "aurora"
+  master_password     = "mustbeeightcharaters"
```

Comment:
Not important, but --
```suggestion
  master_password     = "mustbeeightcharacters"
```

Reply:
There's more of the same below, I assume from copy/paste. :)

Reply:
Good eyes 😎 fixed across the file 👍